lookout/core/garbage_exclusion.py

```diff
@@ -0,0 +1,15 @@
+"""Pattern to filter out garbage files."""
+from importlib import import_module
+from pathlib import Path
+from typing import Iterable
+
+
+def _gather_patterns() -> Iterable[str]:
+    """Return available patterns to filter out garbage files from the langs package."""
+    for path in (Path(__file__).parent / "langs").iterdir():
+        if path.is_dir():
```

Comment:
Since we are using `iterdir()`, do we need to further check `is_dir()`?

Reply:
yes it returns all files in a dir